taxi/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+    country = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=50, unique=True)
+    password = models.CharField(max_length=50)
+
+    def __str__(self):
+        return self.username
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=50)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name='cars'
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name='cars')
+
+    def __str__(self):
+        return f"CAR:{self.model}, manufacturer:{self.manufacturer}, drivers:{self.drivers}"
```

Comment:
In the `__str__` method of the `Car` model, you are trying to print `self.drivers` which is a ManyToMany relation. This will not work as expected. Instead, you should use a loop to print each driver. However, it's not recommended to include all related objects in the `__str__` method. It's better to return only the main information about the object, such as `self.model` in this case.